SecondSight/main.py

```diff
@@ -14,26 +15,30 @@
 import sys
 import argparse
 
-def main_cli():
-    config = SecondSight.config.Configuration()
-    config.set_path('config.json')
-    cameras=[]
-    for cam_config in config.get_value('cameras'):
-        cameras.append(SecondSight.Cameras.Camera(cam_config['port'], cam_config['calibration'], cam_config['pos'], cam_config['role']))
-    app = Flask(__name__)
-    app.cameras=cameras
-    SecondSight.webserver.DEATHSTARE.start(app)
-    SecondSight.webserver.ApriltagAPI.start(app)
-    threading.Thread(target=app.run,kwargs={'host': "0.0.0.0"}).start()
+
+def mainLoop(app):
     lastframetime = 0
     while True:
         newtime = time.time()
         towait = .1 - (newtime - lastframetime)
         if towait > 0:
             time.sleep(towait)
         lastframetime = newtime
-        for cam in cameras:
+        for cam in app.cameras:
             cam.update()
+        app.apriltags = SecondSight.AprilTags.Detector.fetchApriltags(app.cameras)
+
+
+def main_cli():
+    config = SecondSight.config.loadConfig()
+    cameras = SecondSight.Cameras.loadCameras(config)
+    app = SecondSight.webserver.Server.startFlask(cameras)
+    while not config.get_value('cameras'):
+        print('Waiting for cameras to be added to config')
```

Comment:
When I run this without a config file, it prints this to the terminal as fast as possible. We shouldn't ever be printing to the terminal directly, we should use logging